### ISSUE TITLE
controller installation: error restoring bootparams xcat table

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#286]
version:* r3 06c3594
hardware: amaster, controller02

in /var/log/xcat/xcat.log:

```
Command failed: tabrestore /tmp/xcattables/bootparams.csv 2>&1. Error message: Error: DB error  with line 1: .
Error running tabrestore /tmp/xcattables/bootparams.csv, Error: DB error  with line 1: 
```

**Comment by themisa** [comment]
depends of #286

**Comment by themisa** [comment]
Fixed in dtap 7c7b15f. Waiting to confirm in r3

**Comment by themisa** [comment]
fixed in r3_fixes 2b9985b

**Comment by themisa** [comment]
Fixed in r3 7ac7a06
